extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -243,6 +264,22 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     return adClientId;
   }
 
+  /** @override */
+  getSafeframePath() {
+    if (
```

Comment:
Why so many line returns?

Reply:
Just how `gulp lint --fix` change it 🤷‍♂️

Reply:
cleaned it up a bit though